third_party/terraform/resources/resource_compute_instance_template.go

```diff
@@ -888,3 +865,28 @@ func resourceComputeInstanceTemplateDelete(d *schema.ResourceData, meta interfac
 	d.SetId("")
 	return nil
 }
+
+func expandResourceComputeInstanceTemplateScheduling(d *schema.ResourceData, meta interface{}) (*computeBeta.Scheduling, error) {
+	v, ok := d.GetOk("scheduling")
+	if !ok || v == nil {
+		return &computeBeta.Scheduling{
+			OnHostMaintenance: "MIGRATE",
```

Comment:
Can you add a comment explaining why we put the default in the code?

Reply:
GH is not showing outdated for some reason but added a comment